src/obr/signac_wrapper/operations.py

```diff
@@ -55,6 +55,43 @@ def set_entrypoint(self, entrypoint: dict):
         """
         self._entrypoint = entrypoint
 
+    def group_jobs(self, jobs: list[Job], summarize: int = 0) -> dict[str, list[Job]]:
```

Comment:
Please add a docstring. And a unit test if not too complicated.

Reply:
Also, can you split the function up to make easier to understand or add a bit more documentation?